contents/models.py

```diff
@@ -1,3 +1,16 @@
 from django.db import models
 
 # Create your models here.
+
+
+class Contents_table(models.Model):
```

Comment:
모델 클래스에서는 _tagle 같은 명칭은 빼주세요
```suggestion
class Content(models.Model):
```